src/main/api/publication.ts

```diff
@@ -47,7 +47,7 @@ export class PublicationApi {
 
     public async delete(data: any): Promise<void> {
         const { identifier } = data;
-        await this.publicationRepository.delete(identifier);
+        await this.catalogService.deletePublication(identifier);
     }
```

Comment:
I think we should ask @clebeaupin about this code / logic change. I am concerned that there might have been a specific reason to invoke `publicationRepository.delete()` instead of `catalogService.deletePublication()`.

Reply:
@panaC in your estimation, is there a risk of regression because of the switch to a different API? Or do you think this is a safe code / logic change?

Reply:
I think it's a logic change 😐

Reply:
Sure, but do you think it is a *safe* one? :)
There must be a reason why Leo or Cyrille decided to use the original API, instead of the one you are replacing it with. Thus why I am concerned about potential regression bugs. We need to assert that this is a safe change with no nasty side effects.
Ping @clebeaupin @Scarsniik